backend/views/core/invoices/edit.py

```diff
@@ -140,7 +140,7 @@ def edit_invoice(request: HttpRequest, invoice_id):
 
     messages.success(request, "Invoice edited")
 
-    if request.htmx:
+    if request.headers.get("Hx-Request", None):
```

Comment:
This is different functionality though, the htmx module covers more than just Hx-Request

Reply:
You are sure that this line is working properly ? Because as far as I have been checking in Django HttpRequest source code and docs there is no `htmx` method and `mypy` along with `python` are complaining abut lack lack of any reference. And module from *backend/types/htmx.py* have not been used. 

I think we should use then in `edit_invoice` function `HtmxHttpRequest` from *backend/types/htmx.py* instead of regular *HttpRequest* because then it's all good.

Reply:
We're using [Django-htmx](https://pypi.org/project/django-htmx/ ) is a library that includes .htmx in middleware

Reply:
Ok, so I'll use there `# type: ignore`